.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve buggy behavior
+title: "[BUG]"
+labels: ""
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is. Be sure that the behavior you're describing _is_ a bug--garbo may do things you don't understand, but not understanding something isn't a reason to report it. You should also take that advice into your life outside KoL; the post-9/11 security state has taught is to be wary of our friends and neighbors in a way that is ultimately detrimental to society.
```

Comment:
Do we want snark in a template?

Reply:
This isn't intended as snark, but it probably doesn't belong nevertheless.